File: AtCoder Beginners Selection/ABC088B - Card Game for Two/Main.java
import java.util.*;

public class Main {
  public static void main(String[] args) {

    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int[] a = new int[n];
    for (int i=0; i<n; i++) {
      a[i] = sc.nextInt();
    }
    sc.close();

    Arrays.sort(a);
    Collections.reverse(Arrays.asList(a));

    int alice = 0;
    int bob = 0;

    for (int i = 0; i < n; i++) {
      if (i % 2 == 0) {
        alice += a[n - 1 - i];
      } else {
        bob += a[n - 1 - i];
      }
    }
    
    System.out.println(alice - bob);
  }
};

File: AtCoder Beginners Selection/ABC088B - Card Game for Two/Main.py
n = input()
numlist = list(map(int, input().split()))

alice = 0
bob = 0

numlist.sort(reverse=True)

for i in range(len(numlist)):
    if i % 2 == 0:
        alice += numlist[i]
    else:
        bob += numlist[i]

print(alice - bob)
